pipenv/core.py

```diff
@@ -1625,7 +1625,9 @@ def warn_in_virtualenv():
             click.echo(
                 '{0}: Pipenv found itself running within a virtual environment, '
                 'so it will automatically use that environment, instead of '
-                'creating its own for any project.'.format(
+                'creating its own for any project. You can set '
+                'PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that '
```

Comment:
Small request, can you swap this out to a format string and use `crayons.normal('PIPENV_IGNORE_VIRTUALENVS=1', bold=True)` for consistency?